source/_components/sensor.transmission.markdown

```diff
@@ -13,79 +13,5 @@ ha_release: pre 0.7
 ha_iot_class: "Local Polling"
```

Comment:
With the redirects we can now remove the sensor and switch pages completely.

Reply:
Should I removed it from my branch and commit the change?

Reply:
Delete the files on your `next-transmission` branch and then commit it to this PR.